frontend/src/Components/Asigntask.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { FaUserAlt, FaCaretDown } from "react-icons/fa";
+import { FiSearch } from "react-icons/fi";
+
+const Asigntask = () => {
+  const membername = [
+    {
+      name: "Nawadha",
+      img: "https://images.pexels.com/photos/3370931/pexels-photo-3370931.jpeg?auto=compress&cs=tinysrgb&w=600&lazy=load",
+      status: "2 task in progress"
+    },
+    {
+      name: "Pragya",
+      img: "https://images.pexels.com/photos/3370931/pexels-photo-3370931.jpeg?auto=compress&cs=tinysrgb&w=600&lazy=load",
+      status: "1 task in progress"
+    },
+  ]
+
+  return (
+    <>
+      <div className="w-[30%] h-[400px] m-5">
```

Comment:
dont use fix sizes for the container use rem else remove it

Reply:
sure